### PR TITLE
[unittest] Fix some unittest for yocto

### PR DESCRIPTION
**Pull request by anyj0527**
- Set src_iio test with channels 3 -> 2
- Skip file permission test if the user is root
- Make division compatible with python3

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Comment by gichan-jang** [comment]
Q. What's wrong with these in yocto?

**Comment by anyj0527** [comment]
@gichan-jang 
- If user is root, the negative testcase for inaccessible file fails
- It needs more investigation, but I found that in the yocto - qemu environment, the src_iio test keeps failing with 3 channels.
  But it passes with 2 channels.
- And for the python2 v.s. python3 division behavior

File: tests/nnstreamer_plugins/unittest_plugins.cc [hunk 1]
```diff
@@ -4252,6 +4253,10 @@ TEST (test_tensor_filter, framework_auto_ext_tflite_no_permission_n)
   const gchar *fw_name = NULL;
   GstElement *gstpipe;
 
+  /** If the user is root, skip this test */
+  if (getuid () == 0)
```
Comment (dongju-chae) [inline tests/nnstreamer_plugins/unittest_plugins.cc:4257]: `geteuid`?

Comment (anyj0527) [reply tests/nnstreamer_plugins/unittest_plugins.cc:4257]: It says both `getuid` and `geteuid` return `0` for the root. `geteuid` would be more proper?

Comment (dongju-chae) [reply tests/nnstreamer_plugins/unittest_plugins.cc:4257]: There's a scenario `euid` can be zero even if `uid` is not zero, and it also has permissions to behave like a root.

Comment (anyj0527) [reply tests/nnstreamer_plugins/unittest_plugins.cc]: Thanks! You're right. I updated with `geteuid`

**Review by dongju-chae — APPROVED** [review]
LGTM

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**